### ISSUE TITLE
MAUI-check doesn't detect installed tooling, installs it in odd location

### ISSUE THREAD

**Issue by Ghostbird** [Redth/dotnet-maui-check#159]
My `ANDROID_SDK_ROOT` is set to `${HOME}/android-toolchain/sdk`. When I run `maui-check` it doesn't find anything. When I allow it to “fix” things, it creates `${HOME}/Library`, `${HOME}/Library/Android/sdk` instead and installs stuff there, with the same structure that my `${HOME}/android-toolchain/sdk` has.

It's not a big deal, as I can just delete the `${HOME}/Library` folder again to clean it up. However, it also means that the tool fails to meaningfully check my installation.

## System Information

```bash
$ uname -a
Linux gijsbertdebian 5.10.0-19-amd64 #1 SMP Debian 5.10.149-2 (2022-10-21) x86_64 GNU/Linux
$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
$ dotnet --version
6.0.403
$ dotnet workload list

Installed Workload Ids      Manifest Version      Installation Source
---------------------------------------------------------------------
android-33                  32.0.476/6.0.400      SDK 6.0.400        
maui-android                6.0.547/6.0.400       SDK 6.0.400        

Use `dotnet workload search` to find additional workloads to install.
```

**Comment by Ghostbird** [comment]
I'm closing this, as dotnet workload tooling has evolved a lot, and I haven't had a need for maui-check for years now.
